### ISSUE TITLE
复现报错

### ISSUE THREAD

**Issue by liuzimo** [nsk1001/multi-label-analysis#2]
“Unable to get element from the feed as bytes.” 
请问什么情况

**Comment by doudou5628891** [comment]
我这边运行的时候 reports = sess.run(self.reports, feed_dict={self.net.is_training:-1})  这个 报错Fetch argument None has invalid type <class 'NoneType'>

**Comment by codeXlover39** [comment]
@doudou5628891  hi, did you handle this problem and use this model properly ?

**Comment by doudou5628891** [comment]
> @doudou5628891 hi, did you handle this problem and use this model properly ?

yes, I solve this problem. I remember the problem is a parameter of this model was altered in this last version of code. SO, you can focus on the file named run_graph.py line 22 and try to alter 256 to 512. It's been a long time and I‘m not sure whether it will work. GOOD LUCK!

**Comment by codeXlover39** [comment]
@doudou5628891  thank you very much for your help. Have a nice day.
